packages/vsce/src/extension.ts

```diff
@@ -45,11 +47,34 @@ export async function activate(
 
   const disposable = vscode.commands.registerCommand(
     "vsce.helloWorld",
+
     async () => {
       console.log(`Fallback to get profiles`);
-      const session = await getSession("sample");
-      const resp = await Greeting.greet(session);
-      void vscode.window.showInformationMessage(resp.content);
+      const user = (await vscode.window.showInputBox({
+        // Prompt for InputBox for the user to Enter his Name
+        prompt: "Enter Name ",
+        placeHolder: "Enter your name ",
+      })) as string;
+      try {
+        const session = await getSession("sample");
+        const resp = await Greeting.greet(session);
+        // The content called from Zowe Sample API Service Greeting API
+        const data = resp.content;
+        // Extracting "Hello" from the content
+        const subdata = data.substr(0, 5) + data[6];
```

Comment:
this is likely a wrong suggestion, but demonstrates how you might have lengths instead of hard-coded numbers
```suggestion
        const expectedString = "Hello";
        const subdata = data.substr(0, expectedString.length - 1) + data[expectedString.length];
```